### PR TITLE
[ADEN-9816] Clean up InstantGlobals

### PR DESCRIPTION
**Pull request by Loczi94**
Ticket: https://wikia-inc.atlassian.net/browse/ADEN-9816

Description:
We created corresponding variables in ICBM.

### CODE REVIEW

File: app/modules/ads/setup.js [hunk 1]
```diff
@@ -178,10 +180,10 @@ export const adsSetup = {
     context.set('services.confiant.enabled', instantConfig.get('icConfiant'));
     context.set('services.durationMedia.enabled', instantConfig.get('icDurationMedia'));
     context.set('services.krux.enabled', adsContext.targeting.enableKruxTargeting
-      && instantConfig.isGeoEnabled('wgAdDriverKruxCountries') && !instantConfig.get('wgSitewideDisableKrux'));
+      && instantConfig.get('icKrux') && !instantConfig.get('wgSitewideDisableKrux'));
```
Comment (Bielik20) [inline app/modules/ads/setup.js]: What about `wgSitewideDisableKrux`?

**Review by Bielik20 — CHANGES_REQUESTED** [review]

File: app/modules/ads/setup.js (hunk 1 above)
Comment (fraszczakszymon) [reply app/modules/ads/setup.js]: @Loczi94 I believe we can remove this part.
```suggestion
      && instantConfig.get('icKrux'));
```

File: app/modules/ads/setup.js [hunk 2]
```diff
@@ -127,43 +127,45 @@ export const adsSetup = {
...
+    //  was set to 100 since in ad-engine it's being used in getMoatTrackingStatus() function.
+    context.set('options.video.moatTracking.sampling', 100);
     context.set('options.video.iasTracking.enabled', instantConfig.get('icIASVideoTracking'));
 
-    context.set('options.video.playAdsOnNextVideo', instantConfig.isGeoEnabled('wgAdDriverPlayAdsOnNextFVCountries'));
-    context.set('options.video.adsOnNextVideoFrequency', instantConfig.get('wgAdDriverPlayAdsOnNextFVFrequency'));
-    context.set('options.video.isMidrollEnabled', instantConfig.isGeoEnabled('wgAdDriverFVMidrollCountries'));
-    context.set('options.video.isPostrollEnabled', instantConfig.isGeoEnabled('wgAdDriverFVPostrollCountries'));
+    context.set('options.video.playAdsOnNextVideo', !!instantConfig.get('icFeaturedVideoAdsFrequency'));
+    context.set('options.video.adsOnNextVideoFrequency', instantConfig.get('icFeaturedVideoAdsFrequency'));
```
Comment (fraszczakszymon) [inline app/modules/ads/setup.js:159]: @Loczi94 could you revisit the value of `icFeaturedVideoAdsFrequency` in ICBM? Shouldn't we set it to `3`?

File: app/modules/ads/setup.js [hunk 3]
```diff
@@ -190,20 +192,15 @@ export const adsSetup = {
...
-      instantConfig.isGeoEnabled('wgAdDriverMoatTrackingForFeaturedVideoAdditionalParamsCountries'),
-    );
 
     context.set('custom.serverPrefix', utils.geoService.isProperCountry(['AU', 'NZ']) ? 'vm' : 'wka');
 
     context.set('slots.featured.videoAdUnit', context.get('vast.dbNameAdUnitId'));
     context.set('slots.incontent_player.videoAdUnit', context.get('vast.dbNameAdUnitId'));
 
-    context.set('slots.floor_adhesion.disabled', !instantConfig.isGeoEnabled('wgAdDriverMobileFloorAdhesionCountries'));
+    context.set('slots.floor_adhesion.disabled', !instantConfig.get('icFloorAdhesion'));
```
Comment (fraszczakszymon) [inline app/modules/ads/setup.js:206]: I believe this variable is missing in the ICBM

File: app/modules/ads/setup.js [hunk 4]
```diff
@@ -247,7 +243,7 @@ export const adsSetup = {
       });
     }
 
-    if (instantConfig.isGeoEnabled('wgAdDriverOverscrolledCountries')) {
+    if (instantConfig.get('icOverscrolledTracking')) {
```
Comment (fraszczakszymon) [inline app/modules/ads/setup.js:249]: This variable has missing geo configuration in ICBM

File: app/modules/ads/setup.js [hunk 5]
```diff
@@ -284,7 +280,7 @@ export const adsSetup = {
       'slots.incontent_player.insertBeforeSelector',
     ];
 
-    if (instantConfig.isGeoEnabled('wgAdDriverRepeatMobileIncontentExtendedCountries')) {
+    if (instantConfig.get('icRepeatMobileIncontentExtended')) {
```
Comment (fraszczakszymon) [inline app/modules/ads/setup.js:286]: This variable has missing geo configuration in ICBM

File: app/modules/ads/setup.js [hunk 6]
```diff
@@ -301,13 +297,8 @@ export const adsSetup = {
...
     }
 
-    if (instantConfig.isGeoEnabled('wgAdDriverLazyBottomLeaderboardMobileWikiCountries')) {
-      context.set('slots.bottom_leaderboard.insertOnViewportEnter', true);
-    }
-
-
     // Need to be placed always after all lABrador wgVars checks
-    context.set('targeting.labrador', cacheStorage.mapSamplingResults(instantConfig.get('wgAdDriverLABradorDfpKeyvals')));
+    context.set('targeting.labrador', cacheStorage.mapSamplingResults(instantConfig.get('icLABradorGamKeyValues')));
```
Comment (fraszczakszymon) [inline app/modules/ads/setup.js:304]: This variable has missing geo configuration in ICBM

File: app/modules/ads/setup.js (hunk 2 above)
Comment (Loczi94) [reply app/modules/ads/setup.js:159]: `wgAdDriverPlayAdsOnNextFVFrequency` changed value from 1 to 3 after I added `icFeaturedVideoAdsFrequency` to ICBM. I'll update the value.

**Review by fraszczakszymon — APPROVED** [review]

**Review by Bielik20 — APPROVED** [review]

**Pull request merged**